### DISCUSSION TITLE
Adding Rumble + Memory Card Support with Retroscaler 4 in 1 N64 Game Controller Adapter?

### DISCUSSION THREAD

**Discussion by jackiemoonsoon** [darthcloud/BlueRetro#581] [Q&A]
Hi there,
I recently purchased the Retroscaler 4 in 1 N64 Game Controller Adapter. It's working great but it does not seem to have memory card or rumble pak support. Is there a way to flash the firmware and provide support for this?

Here is the one I bought:
https://www.aliexpress.us/item/3256804935793041.html

Thanks in advance.

**Comment by darthcloud** [comment]
Read docs https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual
